MI429/Program.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Abstractions;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.AppConfig;
+
+namespace MI429
```

Comment:
Could this be integrated into an existing console application? Or generalized. All of these add to the build time.

Reply:
@pmaytak can you please point me to one, i know I have seen one but not able to find it

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/tests/devapps/NetCoreTestApp/Program.cs

Reply:
added now, this did not show up in the sln